lib/nylas/message.rb

```diff
@@ -34,7 +35,10 @@ class Message
     has_n_of_attribute :events, :event
     has_n_of_attribute :files, :file
     attribute :folder, :folder
+    attribute :folder_id, :string
+
     has_n_of_attribute :labels, :label
+    has_n_of_attribute :label_ids, :string
```

Comment:
Do we need to add `starred` and `unread` attributes here also? I'm not familiar with the DSL so I'm just looking at the pattern.

Reply:
I am going to check with these attributes and see what's happening. If we don't need it I will remove them from here. Thanks for the point.

Reply:
We do have `started` and `unread` define above the file so we are good. The `label_ids` was the missing one.

Reply:
They are actually here https://github.com/nylas/nylas-ruby/pull/231/files#diff-6027b1238ffa1b9852b7b40018edcbabL31-L32